web/src/components/AccessRuleNameCell.tsx

```diff
@@ -1,20 +1,25 @@
 import { Box, BoxProps, HStack, Text } from "@chakra-ui/react";
 import React from "react";
 import { Link } from "react-location";
+import { useAdminGetAccessRule } from "../utils/backend-client/admin/admin";
 import { useUserGetAccessRule } from "../utils/backend-client/end-user/end-user";
 import { ProviderIcon } from "./icons/providerIcon";
 
 type Props = {
   reason?: string;
   accessRuleId: string;
+  adminRoute: boolean;
```

Comment:
Ideally we'd make this a presentation-only component that doesn't do any data fetching - this prop couples the way that we are fetching the data with the way we're displaying it. Happy with this approach for now though!